listener/events.go

```diff
@@ -55,7 +55,10 @@ func EventsMessageHandler(m kafka.Message) error {
 
 func HandleUpdate(event HostEgressEvent) error {
 	var system models.SystemPlatform
-	if err := database.Db.Find(&system, "inventory_id = ?", event.Host.ID).Error; err != nil {
+	err := database.Db.Find(&system, "inventory_id = ?", event.Host.ID).Error
+	if err != nil && gorm.IsRecordNotFoundError(err) {
+		return nil
```

Comment:
what to log it at least?

Reply:
Thanks. I added metrics to different code paths.